CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ rosidl_generate_interfaces(
   "msg/LEDAnimationQueue.msg"
   "msg/LEDImageAnimation.msg"
   "msg/MotorControllerState.msg"
+  "msg/RobotState.msg"
```

Comment:
I'm not so sure about this name `RobotState` but I cannot find any alternatives for now. 

I just leave it here ;)